cli/cmd/help.go

```diff
@@ -114,10 +115,11 @@ func getExternalCommandsString(modulesInfo *modules.ModulesInfo) string {
 	}
 
 	if str != "" {
+		str = util.Bold("\nEXTERNAL COMMANDS\n") + str
 		return str[:len(str)-1]
 	}
 
-	return fmt.Sprintf("  %s", noAvailableExternalCmdMsg)
+	return fmt.Sprintf("")
```

Comment:
Why do we need Sprintf for an empty string?

Reply:
Removed